docs/operations-guide/running-metabase-on-elastic-beanstalk.md

```diff
@@ -38,95 +77,117 @@ As for the domain URL, Feel free to get creative — just remember that the URL
+Now go to the Capacity section and click `Edit`
+![Elastic Beanstalk Networking configuration](images/EBCapacity.png)
+
+The only change you need to do here is to reduce the number of Instances from 4 (the default number) to 1, as we still haven't created a centralized database where Metabase will save all of its configurations and will be using only the embedded H2 database which lives __inside__ the Metabase container and [is *not recommended* for production workloads](https://www.metabase.com/docs/latest/operations-guide/configuring-application-database.html) as there will be no way to backup and maintain that database. **When your instance is restarted for any reason you'll lose all your Metabase data**. If you are just doing a quick trial of Metabase that may be okay but otherwise you would like to start [creating your database engine in RDS separately](creating-RDS-database-on-AWS.html) or deploy one a separate server.
+
+![Elastic Beanstalk Networking configuration](images/EBCapacityModified.png)
+
+Now click on `Save` at the bottom of the page and you can now click on `Create App` at the end of the Configuration page to start creating the environment.
+
+## Step 3 - Wait for your environment to start
+
+This can take a little while depending on AWS. It’s not uncommon to see this take 10-15 minutes, so feel free to do something else and come back to check on it. What's happening here is each part of the environment is being provisioned with AWS's infrastructure automation functionality named CloudFormation (so you can see the detailed progress for the creation of your environment if you open CloudFormation in another tab).
+
+When it's all done you should see something like this:
+
+![ebcomplete](images/EBComplete.png)
+
+To see your new Metabase instance, simply click on the link under your environment name in the top-left (it will end with `.elasticbeanstalk.com`)
+
+Now that you’ve installed Metabase, it’s time to [set it up and connect it to your database](../setting-up-metabase.md).
 
-Once you've entered a password and clicked `Save`, the red outline on the Database box should have gone away, indicating that the application is valid and ready to be launched.
+# Additional Options
 
-### Additional Options
+There are many ways to customize your Elastic Beanstalk deployment, but commonly modified settings include:
 
-#### Using Metabase in a VPC
+## Instance Details
 
-Newer AWS accounts are encouraging the use of Virtual Private Cloud (VPC) for deployments, and in general we think it's simplest to follow that best practice.
+- `Instance type` (`Instances` block) is for picking the size of AWS instance you want to run. We recommend **at least** `t3a.small` for most uses. You can always [scale](https://www.metabase.com/learn/data-diet/analytics/metabase-at-scale.html) vertically by changing this configuration.
+- `EC2 key pair` (`Security` block) is only needed if you want to SSH into your instance directly which is __not recommended__
```

Comment:
```suggestion
- `EC2 key pair` (`Security` block) is only needed if you want to SSH into your instance directly which is __not recommended__.
```

Did you mean to use italics for emphasis here? Then it should be a single `_` on either side. If you meant for it to be bold, it's probably best to keep it consistent with other styling (`**`).